File: app/src/main/java/org/eclipsesoundscapes/ui/rumblemap/RumbleMapInstructionsActivity.java
package org.eclipsesoundscapes.ui.rumblemap;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ImageButton;
import android.widget.TextView;

import org.eclipsesoundscapes.R;

import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.OnClick;

public class RumbleMapInstructionsActivity extends AppCompatActivity {

    @BindView(R.id.bottom_view_title) TextView title;
    @BindView(R.id.bottom_view_more) TextView instructions;
    @BindView(R.id.exit_button) ImageButton closeButton;

    @OnClick(R.id.exit_button) void onExit(){
        onBackPressed();
    }


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_rumble_map_instructions);
        ButterKnife.bind(this);
    }

    @Override
    public void onBackPressed() {
        super.onBackPressed();
        finish();
        overridePendingTransition(R.anim.anim_slide_in_left, R.anim.anim_slide_out_right);
    }
}

File: app/src/main/java/org/eclipsesoundscapes/ui/rumblemap/RumbleMapInstructionsActivity.kt
package org.eclipsesoundscapes.ui.rumblemap

import android.os.Build
import android.os.Bundle
import androidx.activity.addCallback
import org.eclipsesoundscapes.R
import org.eclipsesoundscapes.databinding.ActivityRumbleMapInstructionsBinding
import org.eclipsesoundscapes.ui.base.BaseActivity

class RumbleMapInstructionsActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = ActivityRumbleMapInstructionsBinding.inflate(layoutInflater).apply {
            exitButton.setOnClickListener { onBackPressed() }
        }
        val view = binding.root
        setContentView(view)

        onBackPressedDispatcher.addCallback {
            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
                overrideActivityTransition(OVERRIDE_TRANSITION_CLOSE, R.anim.anim_slide_in_left, R.anim.anim_slide_out_right)
            } else {
                overridePendingTransition(R.anim.anim_slide_in_left, R.anim.anim_slide_out_right)
            }

            finish()
            return@addCallback
        }
    }
}